packages/react-vapor/src/components/datePicker/DatePickerDropdown.tsx

```diff
@@ -273,24 +275,50 @@ export class DatePickerDropdown extends React.Component<IDatePickerDropdownProps
             : DateUtils.getSimpleDate(date);
     }
 
+    private getLimitAndDiffInMinutes(range: IRangeLimit): {limit: number; diff: number} {
+        const {weeks, days, hours, minutes} = range;
+        const limit: number =
+            (weeks ? weeks * 10080 : 0) + (days ? days * 1440 : 0) + (hours ? hours * 60 : 0) + (minutes ? minutes : 0);

```

Comment:
FYI: You can use `moment.duration({year: 2, weeks: 3}).asMinutes()` to create a duration and get the duration in a specific unit.